modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/tests/HeadlessDeliverySupportDifferentParametersDM.testcase

```diff
@@ -77,7 +77,7 @@ definition {
 
 			TestUtils.assertEquals(
 				actual = ${actualValue},
-				expected = "{title=Document_1.txt}");
+				expected = "{fileName=Document_1.txt, title=Document_1.txt}");
```

Comment:
Hi @marcogalluzzi 
the test (CanReceiveTitleFieldValuesOnlyInResponseForIndividualElement) expects that only the title is changed, but with this changes, 2 attributes are changed.

The system allows to change only the title, and I think that is the meaning of this test.

I do not know if we should change the way we are making the call or change the name of the test or another change.

Reply:
I think we just need to add fileName to the list of restrictFields in https://github.com/liferay-lima/liferay-portal/pull/4232/files/e44500548482998db4f2671e06990487717e9212..b21d8bde165bdde49e935c75c853eec9caf16311#diff-903d61f88dcedb859ef6371e9763088f6fa901cd7b8c9f0b15aff251a95b7832R72 so that only the title shows up